builder/Dockerfile.linux

```diff
@@ -1,11 +1,12 @@
+# Base image depends on target platform.
+# See: https://mcr.microsoft.com/en-us/product/cbl-mariner/distroless/base/about
+ARG BASE_IMAGE=mcr.microsoft.com/cbl-mariner/distroless/base:2.0
```

Comment:
love that the dockerfiles are consistent!

Reply:
They were close enough that I consolidated them into one for a while. But eventually changed my mind and separated them again (they were just different enough for the special cases to be hard to reason about).